Feature/documentation updates

stratkg opened the pull request.

clabornd commented:
Couple references to "Condition" main effect in helper_fn.R examples, imd_anova() has the same old pval_adjust argument in the same examples:

https://github.com/pmartR/pmartR/blob/9d6a4aafa834c22959d4f15802e62114742fb0c2/R/helper_fn.R#L915
https://github.com/pmartR/pmartR/blob/9d6a4aafa834c22959d4f15802e62114742fb0c2/R/helper_fn.R#L975

clabornd commented:
Reference to the smoking mice dataset left in the trelliscope vignette:  https://github.com/pmartR/pmartR/blob/9d6a4aafa834c22959d4f15802e62114742fb0c2/vignettes/Trelliscope_Vignette.Rmd#L225

Just replacing it with pep_edata seems to work.

clabornd commented on vignettes/Quality_Control_with_pmartR.Rmd:
```diff
@@ -10,28 +10,28 @@ output:
 
 library(pmartR)
-library(pmartRdata)
+# library(pmartRdata)
```
uncomment

stratkg replied:
This vignette shouldn't be in the package any more; must've reappeared with PR #228. Removed.

clabornd commented on vignettes/Data_Normalization.Rmd:
````diff
@@ -0,0 +1,385 @@
+We'll use the example NMR dataset from the `pmartRdata` package.
+
+```{r}
+mynmr <- data(nmr_object_identified)
````
This block errors out for me, I think this should be changed to:

```
data(nmr_identified_object)
summary(nmr_identified_object)

# log2 transform the values
nmr_object <- edata_transform(omicsData = mynmr, data_scale = "log2")
```

stratkg replied:
You're correct, good catch!

clabornd commented:
imo we should not commit the .html files as they'll hopefully be committed on another branch that deploys the documentation.  Also wondering if we can delete the data-raw folder.

stratkg commented:
> Couple references to "Condition" main effect in helper_fn.R examples, imd_anova() has the same old pval_adjust argument in the same examples:
> 
> https://github.com/pmartR/pmartR/blob/9d6a4aafa834c22959d4f15802e62114742fb0c2/R/helper_fn.R#L915
> 
> 
> https://github.com/pmartR/pmartR/blob/9d6a4aafa834c22959d4f15802e62114742fb0c2/R/helper_fn.R#L975

Updated to 'Phenotype' for new proData object, and removed the "dontrun" around those examples

stratkg commented:
> Reference to the smoking mice dataset left in the trelliscope vignette:
> 
> https://github.com/pmartR/pmartR/blob/9d6a4aafa834c22959d4f15802e62114742fb0c2/vignettes/Trelliscope_Vignette.Rmd#L225
> 
> Just replacing it with pep_edata seems to work.

Good catch. Also replaced "Mass_Tag_ID" with "Peptide" in that code chunk.

stratkg commented:
> imo we should not commit the .html files as they'll hopefully be committed on another branch that deploys the documentation. Also wondering if we can delete the data-raw folder.

Sounds good to me, I've removed the .html's of the vignettes. Double checking with folks about the data-raw folder, but I imagine we don't need it. More later.

stratkg commented:
> > imo we should not commit the .html files as they'll hopefully be committed on another branch that deploys the documentation. Also wondering if we can delete the data-raw folder.
> 
> Sounds good to me, I've removed the .html's of the vignettes. Double checking with folks about the data-raw folder, but I imagine we don't need it. More later.

Removing the data-raw folder.

clabornd reviewed (approved):
One vignette failed, but it's not related to this PR.  Some plot functions are broken due to patchwork not having this import statement in NAMESPACE:  importFrom(patchwork, plot_layout).  It was removed in 56130d0cad9562b8ba19fe67a9fd3018fb13f1b7

clabornd commented on vignettes/Data_Normalization.Rmd:
````diff
@@ -0,0 +1,384 @@
+
+
+```{r spans plot, out.width = "600px", echo=FALSE}
+knitr::include_graphics("SPANS_pep.png")
````
Vignette build fails for me here since knitr uses the .Rmd file location as its root.  I think we want to move SPANS_pep.png into the vignettes folder.

stratkg replied:
You're right, done.

clabornd commented on vignettes/Data_Normalization.Rmd:
````diff
@@ -0,0 +1,384 @@
+
+
+```{r}
+head(spans_result)
````
`spans_result` doesn't exist during build, I think we needed to load `spans_result.RDS` before here.  The committed spans_result.RDS didn't match the results shown in SPANS_pep.png but re-running SPANS gives the same result as the .png - I have an updated spans_result.RDS in the github actions pull request.

clabornd replied:
Ah I see, the spans_result.RDS looks like the one for isobaric pepdata.

stratkg replied:
Didn't do anything here...seemed resolved?

clabornd replied:
sry didn't mean to imply this was resolved, see my comment in the new PR

clabornd commented on vignettes/Trelliscope_Vignette.Rmd:
````diff
@@ -84,7 +84,7 @@ Let's start with the simplest input data to run these trelliscope functions,
 
 ```{r, echo = F}
-datatable(head(pmartRdata::lipid_edata_pos), options = list(`scrollX` = T))
+datatable(head(pmartRdata::lipid_pos_edata), options = list(`scrollX` = T))
````
fails since the DT library is not a dependency I think.  I think we can change datatable -> knitr::kable()

stratkg replied:
Good idea! Done.

stratkg commented:
> > > imo we should not commit the .html files as they'll hopefully be committed on another branch that deploys the documentation. Also wondering if we can delete the data-raw folder.
> > 
> > 
> > Sounds good to me, I've removed the .html's of the vignettes. Double checking with folks about the data-raw folder, but I imagine we don't need it. More later.
> 
> Removing the data-raw folder.

Removed the html vignette files as well
